backend/prompt_studio/prompt_studio_index_manager/prompt_studio_index_helper.py

```diff
@@ -1,10 +1,10 @@
 import json
 import logging
-from typing import Optional
 
 from prompt_studio.prompt_profile_manager.models import ProfileManager
 from prompt_studio.prompt_studio_document_manager.models import DocumentManager
 
+from ..prompt_studio_core.exceptions import IndexingAPIError
```

Comment:
@chandrasekharan-zipstack will relative import cause any issue? I remember you mentioning something. If yes, is there anyother way to raise an exception here?